src/messages/MOSDECSubOpReadReply.h

```diff
@@ -63,7 +68,12 @@ class MOSDECSubOpReadReply : public MOSDFastDispatchOp {
     using ceph::encode;
     encode(pgid, payload);
     encode(map_epoch, payload);
-    encode(op, payload);
+    if (!HAVE_FEATURE(features, SERVER_SQUID)) {
```

Comment:
Reviewer feedback requested:

The PR is currently using the feature SERVER_SQUID to select new/old version encoding for MOSDECSubOpReadReply, MOSDECSubOpWrite and MOSDRepOp. Should this be SERVER_TENTACLE? Is there a better way of checking compatibility of remote OSDs?